daemons/fenced/fenced_commands.c

```diff
@@ -2977,8 +2983,11 @@ handle_request(pcmk__client_t *client, uint32_t id, uint32_t flags,
                    crm_element_value(dev, F_STONITH_ACTION),
                    crm_element_value(dev, F_STONITH_TARGET));
 
-        if (initiate_remote_stonith_op(NULL, request, FALSE) != NULL) {
+        if (initiate_remote_stonith_op(NULL, request, FALSE) == NULL) {
+            rc = -EPROTO;
```

Comment:
Changing the rv from -EOPNOTSUP to -EPROTO sounds as if it makes sense but opposed to description it changes the behavior ;-)

Reply:
True ;) but it shouldn't ever happen in practice. It just makes a later change easier ...